.github/workflows/name: "Python lint and test"  on:   push:     workflow_dispatch:  jobs:   build:     runs-on: "self-hosted"     strategy:       matrix:         python-version: ["3.12"]      steps:       - uses: "actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744" # ratchet:actions/checkout@v3       - name: "Set up Python ${{ matrix.python-version }}"         uses: "actions/python_selfhost.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Python lint and test"
```

Comment:
nit: we generally prefer single quotes for yaml strings. I appreciate you quoting your strings though as most people don't and it can lead to [absolute chaos](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)!

Reply:
Oops maybe I shouldn't have had auto-merge enabled so I could change that. Noted for the future!